### ISSUE TITLE
Cannot delete a resource in live demo

### ISSUE THREAD

**Issue by sh41** [api-platform/demo#16]
Steps to reproduce:

```
$ curl -D - -s -X DELETE "https://demo.api-platform.com/books/2" -H  "accept: application/json"
HTTP/2 500
date: Sun, 30 Jul 2017 11:06:12 GMT
content-type: application/problem+json; charset=utf-8
content-length: 122
set-cookie: __cfduid=d074e5de5bce897bc9de63740d76736a71501412772; expires=Mon, 30-Jul-18 11:06:12 GMT; path=/; domain=.api-platform.com; HttpOnly
x-powered-by: PHP/7.1.2
x-content-type-options: nosniff
x-frame-options: deny
cache-control: no-cache, private
link: <http://demo.api-platform.com/docs.jsonld>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
x-varnish: 988692
age: 0
via: 1.1 varnish-v4
server: cloudflare-nginx
cf-ray: 3867d8e6d9c50a6c-LHR

{"type":"https:\/\/tools.ietf.org\/html\/rfc2616#section-10","title":"An error occurred","detail":"Internal Server Error"}
```
Expected behaviour: 200 OK response and the resource is deleted.

**Comment by sh41** [comment]
Seems like this is not limited to DELETE method:
```
$ curl -s -D - -X PUT "https://demo.api-platform.com/books/2" -H  "accept: application/ld+json" -H  "content-type: application/ld+json" -d "{  \"@context\": \"/contexts/Book\",  \"@id\": \"/books/2\",  \"@type\": \"http://schema.org/Book\",  \"id\": 2,  \"isbn\": \"9790070863971\",  \"description\": \"Bidon, ce livre n'existe pas. Bref c'est un fake ...\",  \"author\": \"Virginia Dooley\",  \"title\": \"X\",  \"publicationDate\": \"2002-01-29T00:00:00+00:00\"}"
HTTP/2 500
date: Sun, 30 Jul 2017 11:15:51 GMT
content-type: application/ld+json; charset=utf-8
content-length: 132
set-cookie: __cfduid=d6ce5a057e608e976350bdae8953574641501413351; expires=Mon, 30-Jul-18 11:15:51 GMT; path=/; domain=.api-platform.com; HttpOnly
x-powered-by: PHP/7.1.2
x-content-type-options: nosniff
x-frame-options: deny
cache-control: no-cache, private
link: <http://demo.api-platform.com/docs.jsonld>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
x-varnish: 599359
age: 0
via: 1.1 varnish-v4
server: cloudflare-nginx
cf-ray: 3867e7051d380a6c-LHR

{"@context":"\/contexts\/Error","@type":"hydra:Error","hydra:title":"An error occurred","hydra:description":"Internal Server Error"}
```

```
$ curl -D - -s -X POST "https://demo.api-platform.com/books" -H  "accept: application/ld+json" -H  "content-type: application/ld+json" -d "{      \"isbn\": \"9790070863971\",      \"description\": \"Bidon, ce livre n'existe pas. Bref c'est un fake ...\",      \"author\": \"Virginia Dooley\",      \"title\": \"X\",      \"publicationDate\": \"2002-01-29T00:00:00+00:00\"    }"
HTTP/2 500
date: Sun, 30 Jul 2017 11:19:00 GMT
content-type: application/ld+json; charset=utf-8
content-length: 132
set-cookie: __cfduid=ddc7b3da8d277bc231d042e21da8349fc1501413540; expires=Mon, 30-Jul-18 11:19:00 GMT; path=/; domain=.api-platform.com; HttpOnly
x-powered-by: PHP/7.1.2
x-content-type-options: nosniff
x-frame-options: deny
cache-control: no-cache, private
link: <http://demo.api-platform.com/docs.jsonld>; rel="http://www.w3.org/ns/hydra/core#apiDocumentation"
x-varnish: 599382
age: 0
via: 1.1 varnish-v4
server: cloudflare-nginx
cf-ray: 3867eba2ec1613a7-LHR

{"@context":"\/contexts\/Error","@type":"hydra:Error","hydra:title":"An error occurred","hydra:description":"Internal Server Error"}
```

**Comment by dunglas** [comment]
Should be fixed by #24, thanks.
